friendly-frogs/friendly_frogs/users/views.py

```diff
@@ -1,15 +1,23 @@
 from django.shortcuts import render
-from .forms import UserRegisterForm
+from .forms import UserRegisterForm, ProfileForm
 from django.contrib import messages
 
 
 def register(request):
     if request.method == "POST":
         form = UserRegisterForm(request.POST)
-        if form.is_valid():
+        form_p = ProfileForm(request.POST, instance=request.user.profile)
+        if form.is_valid() and form_p.is_valid():
             form.save()
+            form_p.save()
             username = form.cleaned_data.get('username')
-            messages.success(request, f"Account successfully created for {username}")
+            messages.success(
+                request, f"Account successfully created for {username}"
+                )
     else:
         form = UserRegisterForm()
-    return render(request, "users/register.html", {'form': form})
+        form_p = ProfileForm
+    context = {'form': form, 'form_p': form_p}
+    return render(
+        request, "users/register.html", context
+                )
```

Comment:
Looks like this could all be on the same line, same for a few more hanging indents in the files in the PR like on lines 14-16 in this file.